api/src/routes/stardust/nftDetails/get.ts

```diff
@@ -1,11 +1,62 @@
 import { ServiceFactory } from "../../../factories/serviceFactory";
 import { INftDetailsRequest } from "../../../models/api/stardust/INftDetailsRequest";
-import { INftOutputsResponse } from "../../../models/api/stardust/INftOutputsResponse";
+import { INftDetailsResponse } from "../../../models/api/stardust/INftDetailsResponse";
 import { IConfiguration } from "../../../models/configuration/IConfiguration";
 import { STARDUST } from "../../../models/db/protocolVersion";
 import { NetworkService } from "../../../services/networkService";
-import { StardustTangleHelper } from "../../../utils/stardust/stardustTangleHelper";
 import { ValidationHelper } from "../../../utils/validationHelper";
+import "dotenv/config";
+
+/**
+ * Get mock nft detils
+ * @returns nft details.
+ */
+ export function getMockNftDetails(): INftDetailsResponse {
```

Comment:
We don't need to export this. Actually it should be `private` and possibly `readonly`.

Reply:
Done